Meta/CMake/code_generators.cmake

```diff
@@ -51,6 +51,8 @@ function(compile_ipc source output)
     endif()
     add_custom_command(
         OUTPUT ${output}
+        get_filename_component(output_path ${output} DIRECTORY)
```

Comment:
wait. it's not valid to do this here, is it? you would need to do this outside of the `add_custom_command` block.

At any rate, ninja is very upset with this change, does it work for you locally? https://github.com/SerenityOS/serenity/actions/runs/7952043992/job/21706189842?pr=23250#step:25:52